src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -198,6 +204,10 @@ public String getReplacementStrategy() {
         return this.replacementStrategy;
     }
 
+    public boolean getTaggable() {
```

Comment:
doesn't lombok handle this? isTaggable()?

Reply:
Yes, lombok generated `isTaggable` for it. Thanks.